### PR TITLE
gui-apps/waybar: add 0.11.0

### PR DESCRIPTION
**Pull request by quincyf467**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

**Comment by quincyf467** [comment]
@fkobi Okay. no problem.

**Comment by quincyf467** [comment]
@fkobi okay

**Review by MrRoy — APPROVED** [review]
PR looks good to me, thanks!

**Comment by MrRoy** [comment]
Commit history looks fine to me as it is.

File: gui-apps/waybar/waybar-0.11.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+DESCRIPTION="Highly customizable Wayland bar for Sway and Wlroots based compositors"
+HOMEPAGE="https://github.com/Alexays/Waybar"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Alexays/${PN^}.git"
+else
+	SRC_URI="https://github.com/Alexays/${PN^}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="amd64 ~arm64"
```
Comment (ceamac) [inline gui-apps/waybar/waybar-0.11.0.ebuild]: `ekeyword ~all waybar-0.11.0.ebuild`

**Review by ceamac — COMMENTED** [review]
Looks good, but please sync the live ebuild too

File: gui-apps/waybar/waybar-0.11.0.ebuild (hunk 1 above)
Comment (quincyf467) [reply gui-apps/waybar/waybar-0.11.0.ebuild]: okay. thanks for pointing that out.

Comment (ceamac) [reply gui-apps/waybar/waybar-0.11.0.ebuild]: Can you try to squash this commit with the one adding 0.11.0?  No point in adding it in 2 commits

**Comment by ceamac** [comment]
And also sync the live version (i.e., waybar-9999 should be as close as possible to waybar-0.11.0).  Live ebuilds tend to fall behind.

**Comment by quincyf467** [comment]
done

File: gui-apps/waybar/waybar-0.11.0.ebuild (hunk 1 above)
Comment (quincyf467) [reply gui-apps/waybar/waybar-0.11.0.ebuild]: okay

**Review by ceamac — APPROVED** [review]
Great!  Thank you.

I'll reformat the commit messages a bit, there are a few pkgcheck warnings.

**Pull request closed without merging**